clusterdef/nodegroup.go

```diff
@@ -21,6 +21,10 @@ type DockerNodeGroup struct {
 
 type CloudNodeGroup struct {
 	InstanceType string `yaml:"instance-type,omitempty"`
+	Cpu          int    `yaml:"cpu,omitempty"`
+	MemoryInGB   int    `yaml:"memory-in-gb,omitempty"`
```

Comment:
The "InGB" portion of this doesn't seem to align with the naming of other fields I don't think.  Additionally, I believe that the cluster definition actually specifies the memory requirements for each service, which aggregated can be used to imply the required size of the instances.

Reply:
Updated